picasso-pollexor/src/test/java/com/squareup/picasso/pollexor/PollexorRequestTransformerTest.java

```diff
@@ -1,19 +1,24 @@
 package com.squareup.picasso.pollexor;
 
 import android.net.Uri;
-import com.squareup.picasso.Request;
+import com.squareup.picasso.*;
 import com.squareup.pollexor.Thumbor;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.robolectric.RobolectricGradleTestRunner;
 import org.robolectric.RobolectricTestRunner;
 import org.robolectric.annotation.Config;
 
 import static com.squareup.picasso.Picasso.RequestTransformer;
 import static org.fest.assertions.api.Assertions.assertThat;
 import static org.robolectric.annotation.Config.NONE;
 
-@RunWith(RobolectricTestRunner.class) //
-@Config(manifest = NONE)
+@RunWith(RobolectricGradleTestRunner.class)
+@Config(
+    sdk = 18,
```

Comment:
Robolectric is dumb and will try to use your min SDK for this otherwise, even though robolectric itself only supports 18+. Do without this, it yells at you for using an unsupported API version